bluetooth-printer/index.html

```diff
@@ -25,6 +25,8 @@
     <meta name="mobile-web-app-capable" content="yes">
     <meta name="apple-mobile-web-app-capable" content="yes">
 
+    <meta name="origin-trials" content="1|M3SuHper0XloxgNIpwzxzeimc1a9HgekQzz7wapb7rnvryOQCdoZMiNL3KHD999xMjt7MY2dOqLV9nmm8oQkCA==|https://webbluetoothcg.github.io:443|WebBluetooth|1460383093">
```

Comment:
[Discussion on this is ongoing](https://groups.google.com/a/chromium.org/d/msgid/experimentation-dev/CA%2BOSsVYS8kLduKy%3D%2B7AGYwRg2gPnqrKsAgZond_07Yt%3DVFe2iA%40mail.gmail.com?utm_medium=email&utm_source=footer) on the experimentation-dev mailing list. As of today, it's still `<meta name="orgin-trials"...` though.